oi/OICodeGen.h

```diff
@@ -51,18 +51,25 @@ struct ParentMember {
 class OICodeGen {
  public:
   struct Config {
+    Config() = default;
+    Config(const Config& other) = delete;
+    Config& operator=(const Config& other) = delete;
+    Config(Config&& other) = default;
+    Config& operator=(Config&& other) = default;
+
     /*
      * Note: don't set default values for the config so the user gets an
      * uninitialized field" warning if they missed any.
      */
```

Comment:
@ttreyer I don't understand what exactly this comment is getting at. I've tried a few things and can't seem to get an uninitialised field warning